release_notes/op-release-notes-1-17.adoc

```diff
@@ -31,6 +31,6 @@ include::modules/op-tkn-pipelines-compatibility-support-matrix.adoc[leveloffset=
 // Making open source more inclusive
 include::modules/making-open-source-more-inclusive.adoc[leveloffset=+1]
```

Comment:
I do think the release-notes-1.16 file should be kept in main even though it is no longer displayed